### ISSUE TITLE
Support disabling/enabling inspections by name

### ISSUE THREAD

**Issue by wolfs** [JetBrains/gradle-idea-ext-plugin#41]
It should be possible to enable/disable inspections by name via the `idea-ext` plugin. It would be nice to change the severity.

The current workaround we use is to check-in the `.idea/inspectionProfiles` directory.

**Comment by trejkaz** [comment]
Yes please. Does anyone know if there is a good way to do this in the meantime? I'm thinking copying the XML file out into a different location and having the build copy it back into place, except that I was never able to get hooks working, so I'm not sure how I can actually do the copy.

**Comment by Vampire** [comment]
What's the status here?
I see an API though it is not documented and it seems to not work either.
Or I just used it wrongly.
